app/assets/javascripts/discourse/controllers/preferences/account.js.es6

```diff
@@ -54,6 +55,40 @@ export default Ember.Controller.extend(
       );
     },
 
+    @computed("model.associated_accounts")
+    associatedAccountsLoaded(associatedAccounts) {
+      return associatedAccounts !== undefined;
```

Comment:
I think it's better to check against typeof here (https://stackoverflow.com/questions/3390396/how-to-check-for-undefined-in-javascript)

```
return type associatedAccounts !== "undefined";
```